sdk/keyvault/azure-security-keyvault-keys/test/ut/test_consts.hpp

```diff
@@ -0,0 +1,154 @@
+// cspell:disable
```

Comment:
Did you recreate the recordings after updating the API surface?

Reply:
those that needed recreating, yes

Reply:
most of the 68 files are .json , those are the recordinggs